### ISSUE TITLE
Port to latest version of TypeScript

### ISSUE THREAD

**Issue by garycourt** [evanw/typescript-closure-compiler#5]
Have you looked into what would be involved in porting this project to the latest version of TypeScript?

**Comment by evanw** [comment]
Not really. It turns out that what I really wanted was something like [Skew](https://github.com/evanw/skew) so I've been working on that instead.

**Comment by marty-krishner** [comment]
Can you please do it at least once.

**Comment by evanw** [comment]
Nope, sorry. I don't have a need for something like this right now so I'm not going to build it.

**Comment by sagifogel** [comment]
Hi Evan,

My name is Sagi.
I've created a similar project on top of TypeScript 1.7.5
[https://github.com/sagifogel/typescript-closure-compiler](https://github.com/sagifogel/typescript-closure-compiler)
Is it possible for you to add a link to my repository in your README.md file and/or in the description of your npm?

Thanks.

**Comment by marty-krishner** [comment]
Great man.

Keep it updated, it would be really gives benefits to many typescript
lovers.

On Tue, Mar 29, 2016 at 6:19 PM, Sagi <notifications@github.com> wrote:

> Hi Evan,
>
> My name is Sagi.
> I've created a similar project on top of TypeScript 1.7.5
> https://github.com/sagifogel/typescript-closure-compiler
> Is it possible for you to add a link to my repository in your README.md
> file and/or in the description of your npm?
>
> Thanks.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/evanw/typescript-closure-compiler/issues/5#issuecomment-202877423>
>



-- 
--

Thanks & Regards

*- Marty Krishner*
- Skype: marty.krishner
- marty.krishner@gmail.com

**Comment by evanw** [comment]
@sagifogel I can definitely do that. I think it might make more sense just to take down this repo and hand the npm package over to you. What do you think? This repo is so out of date that I don't think it'll be useful to anyone so it doesn't make sense to me to have it around at all.

**Comment by sagifogel** [comment]
@evanw I think its a great  idea. 
You can send me an email (it's available on my github profile) with the details.
